### PR TITLE
Create windows install workflow

### PR DESCRIPTION
**Pull request by fungjj92**
## Overview

Use this pipeline to install the app to the Windows computer. For app updates we'll probably send the zip file to the museum tech consultants, Greenhouse, but may also have to run over to the museum to do an install.

Connects #76

## Testing Instructions

Run `./scripts/cipublish` locally from within the VM and see the project zips up all its built assets and the app runs correctly from the `index.html` file. 

Currently a stale version is hosted on the Windows computer from `localhost` root and it'll be updated when we're ready to ship! 🚢🚀

### CODE REVIEW

File: scripts/cipublish [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+
+    # Ensure zip is available
+    sudo apt-get install zip
```
Comment (caseycesari) [inline scripts/cipublish]: Consider installing this via an Ansible role. The parcel viewer does this, for example. https://github.com/azavea/pwd-parcel-viewer/tree/develop/deployment/ansible/roles

File: scripts/cipublish [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    # Build static files
+    docker-compose \
+        -f docker-compose.yml \
+        run --rm --no-deps app \
+        yarn build
+
+    # Zip the static files to the top level dir
+    cd "${PWD}"/src/app/build
+    zip -r ../../../fishway.zip ./*
```
Comment (caseycesari) [inline scripts/cipublish]: Consider storing this in `dist`, which is used for other built output.

**Review by caseycesari — CHANGES_REQUESTED** [review]
Working well, but I think the `zip` install needs to be moved to Ansible to ensure if we move the build to CI, the script won't be halted by problems installing it via `apt-get` (like waiting for the user to type `Y` to verify the install, or problems with `sudo`)

File: scripts/cipublish (hunk 1 above)
Comment (fungjj92) [reply scripts/cipublish]: Ah, this is good advice!

**Comment by fungjj92** [comment]
I've made the suggested changes and tested the zip output on the Windows machine -- thanks for those!

**Comment by caseycesari** [comment]
Taking another look.

**Review by caseycesari — APPROVED** [review]
Looks good!

**Comment by fungjj92** [comment]
Thank you for review! Merged.

**Pull request merged**